plugins/sudo/sudo.plugin.sh

```diff
@@ -1,11 +1,21 @@
 #! bash oh-my-bash.module
 
 function _omb_plugin_sudo_add_sudo {
+  function __omb_plugin_sudo_is_inserted {
+    [[ "${READLINE_LINE::4}" = "sudo" ]]; return $?
+  }
   if [[ -z $READLINE_LINE ]]; then
     READLINE_LINE="$(fc -ln -1 | command sed 's/^[[:space:]]\{1,\}//')"
+    ((READLINE_POINT += ${#READLINE_LINE}))
+    __omb_plugin_sudo_is_inserted && return
+  fi
+  if __omb_plugin_sudo_is_inserted; then
+    READLINE_LINE="${READLINE_LINE//sudo /}"
```

Comment:
This will remove every substring `sudo ` in any part of the command line.

```suggestion
    READLINE_LINE="${READLINE_LINE/#sudo }"
```

Also, when the command line looks e.g. `sudoedit file` and when `_is_inserted` succeeds, this fails to remove the prefix `sudo ` (because the space is not present) and just moves the cursor backward by 5 characters.

There are two options. One is to make `_is_inserted` to detect `sudo ` exactly (including the space). Another is to add an additional test to determine whether to remove `sudo `.